blazer/config/application.rb

```diff
@@ -45,6 +45,7 @@ class Application < Rails::Application
     config.log_level = ENV["LOG_LEVEL"].present? ? ENV["LOG_LEVEL"].to_sym : :warn
     config.secret_key_base = ENV["SECRET_KEY_BASE"] || SecureRandom.hex(30)
     config.public_file_server.enabled = ENV["RAILS_SERVE_STATIC_FILES"] != "disabled"
+    config.action_mailer.default_url_options = {host: "localhost:8080"}
```

Comment:
this doesnt need to be an env variable?

Reply:
Well for now everyone runs this locally on localhost:8080, but I suppose it could be. Want me to change it?

Reply:
Done.  I've added a default to localhost just in case the env var isn't present.